libraries/provider_hab_package.rb

```diff
@@ -110,9 +110,7 @@ def strip_version(name)
         def platform_target
           if platform_family?('windows')
             'target=x86_64-windows'
-          elsif platform_family?('rhel') && node['platform_version'].to_i < 6
```

Comment:
Thanks. This was some hot nonesense here

Reply:
It was causing some serious crazy to happen